src/coreclr/jit/instr.h

```diff
@@ -375,6 +375,8 @@ enum insScalableOpts : unsigned
     INS_SCALABLE_OPTS_LSL_N,               // Variants with a LSL #N (eg {<Zt>.<T>}, <Pg>, [<Xn|SP>, <Xm>, LSL #2])
     INS_SCALABLE_OPTS_MOD_N,               // Variants with a <mod> #N (eg {<Zt>.S }, <Pg>, [<Xn|SP>, <Zm>.S, <mod> #2])
 
+    INS_SCALABLE_OPTS_WITH_VECTOR_PAIR,    // Variants with {<Zn1>.<T>, <Zn2>.<T>} sve register pair (eg splice)
```

Comment:
Not needed for this PR, but if we start to rack up a lot of `insScalableOpts`, we can probably combine this one with `INS_SCALABLE_OPTS_WITH_PREDICATE_PAIR` into something like `INS_SCALABLE_OPTS_WITH_REGISTER_PAIR`.